ngraph/frontend/onnx_import/src/op/org.openvinotoolkit/experimental_detectron/prior_grid_generator.hpp

```diff
@@ -0,0 +1,59 @@
+//*****************************************************************************
+// Copyright 2017-2021 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#pragma once
+
+#include "ngraph/node.hpp"
+#include "ngraph/op/experimental_detectron_prior_grid_generator.hpp"
+#include "onnx_import/core/node.hpp"
+
+namespace ngraph
+{
+    namespace onnx_import
+    {
+        namespace op
+        {
+            namespace set_1
+            {
+                OutputVector experimental_detectron_prior_grid_generator(const Node& node)
+                {
+                    using PriorGridGenerator =
+                        ngraph::op::v6::ExperimentalDetectronPriorGridGenerator;
+
+                    auto inputs = node.get_ng_inputs();
+                    auto priors = inputs[0];
+                    auto feature_map = inputs[1];
+                    auto im_data = inputs[2];
+
+                    PriorGridGenerator::Attributes attrs{};
+                    attrs.flatten =
+                        static_cast<bool>(node.get_attribute_value<int64_t>("flatten1"));
+                    attrs.h = node.get_attribute_value<int64_t>("h");
+                    attrs.w = node.get_attribute_value<int64_t>("w");
+                    attrs.stride_x = node.get_attribute_value<float>("stride_x");
+                    attrs.stride_x = node.get_attribute_value<float>("stride_y");
```

Comment:
please fill the default values based on model-optimizer/extensions/front/onnx/priorgridgenerator_ext.py